jenkins/config.sh

```diff
@@ -0,0 +1,18 @@
+echo "Running on host $(hostname)"
```

Comment:
To me, the split between `config.sh` and `setup.sh` looks rather random to me. What’s the idea here?

Reply:
The main idea was to remove the duplicated explicit passes in plot_full_comparison.sh and setup.sh. But in the normal cases plot_full_comparison is probably called from other scripts that already include setup.sh. Should it also be callable standalone?

Reply:
Looks like plot_full_comparison is actually called on its own. But so we probably can just source the full setup.sh also in there? Should not hurt I guess and then all config stuff is really inside one file.